### PR TITLE
Docs/robot integers

### PR DESCRIPTION
**Pull request by jjo93sa**
Add information about how to force Robot to treat parameters as numbers

### CODE REVIEW

File: docs/deployment/testing.md [hunk 1]
```diff
@@ -264,6 +264,15 @@ generate_tests_caas_test_case_{cluster_type}_service_{service_name}_expected_tit
 generate_tests_caas_test_case_{cluster_type}_verify_timeout: "45 minutes"
 ```
 
+!!! tip "Coerce a value to something other than a string"
```
Comment (mkjpryor) [inline docs/deployment/testing.md:267]: Needs an extra space between the `tip` and the title otherwise it won't render correctly

**Review by mkjpryor — CHANGES_REQUESTED** [review]
Let's use "Robot Framework" instead of "Robot" throughout I think. And also, just "Robot Framework", not "the Robot Framework".

Other than that, it looks good.

File: docs/deployment/testing.md (hunk 1 above)
Comment (jjo93sa) [reply docs/deployment/testing.md]: Done

**Review by jjo93sa — COMMENTED** [review]
Changes made as requested

**Pull request merged**